full-service/src/service/sync.rs

```diff
@@ -147,7 +173,7 @@ pub fn sync_account_next_chunk(
     conn: Conn,
     account_id_hex: &str,
     logger: &Logger,
-) -> Result<(), SyncError> {
+) -> Result<(usize, usize), SyncError> {
```

Comment:
🚧 Under Construction 🚧 

Using usize right now because we're working with lengths of vectors, but also because usize = u32, and that should be safe sending as numbers through reqwest body

Reply:
Moved to AtomicUsize